vignettes_src/orderly.Rmd

```diff
@@ -0,0 +1,104 @@
+---
+title: "Using didehpc to run orderly tasks"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{Using didehpc to run orderly tasks}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+<!-- HEADER -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  error = FALSE,
+  comment = "#>"
+)
+orderly_root <- orderly:::prepare_orderly_example("demo")
+```
+
+When using [orderly](https://github.com/vimc/orderly) some tasks might be slow to run and would be better suited to running on a remote machine. You can use the cluster and [orderly bundles](https://www.vaccineimpact.org/orderly/articles/bundles.html) to achieve this. If your orderly instance has an [OrderlyWeb](https://github.com/vimc/orderly-web) remote then you should favour running tasks on the configured remote via `orderly::orderly_run_remote` (unless it is too slow even for the OrderlyWeb remote). If your orderly instance is using a [sharepoint remote](https://github.com/vimc/orderly.sharepoint) or has no remote then you can use the cluster to run your slow running tasks.
```

Comment:
I think the "should favour" here is too strong - it will depend on the installation, and the cluster will typically have more power than a node that we run OW on